src/gcs-blobstore-backup-restore/contract_test/bucket_test.go

```diff
@@ -213,7 +213,7 @@ var _ = Describe("Bucket", func() {
 				backupsToComplete, err := gcs.BuildBackupsToComplete(MustHaveEnv("GCP_SERVICE_ACCOUNT_KEY"), config)
 				Expect(err).NotTo(HaveOccurred())
 				_, err = backupsToComplete["droplets"].BucketPair.LiveBucket.ListBlobs("")
-				Expect(err).To(MatchError("storage: bucket doesn't exist"))
+				Expect(err).To(MatchError("storage: bucket doesn't exist: googleapi: Error 404: The specified bucket does not exist., notFound"))
```

Comment:
Might be worth putting another matcher in here to guard against future changes:

```go
Expect(err).To(MatchError(ContainSubstring("storage: bucket doesn't exist")))
```

Reply:
made the change as suggested